src/FieldThumbnailBrowser.h

```diff
@@ -20,40 +20,36 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include "calchartdoc.h"
-
-#include <wx/dialog.h>
 #include <wx/docview.h>
 
-class FieldBrowserView;
-class FieldBrowserPerCont;
+class CalChartDoc;
+class FieldThumbnailBrowserView;
+class FieldThumbnailBrowserPerCont;
 
-class FieldBrowser : public wxScrolledWindow {
+class FieldThumbnailBrowser : public wxScrolledWindow {
     using super = wxScrolledWindow;
 
 public:
-    FieldBrowser(CalChartDoc* dcr, wxWindow* parent, wxWindowID id = wxID_ANY,
+    FieldThumbnailBrowser(CalChartDoc* dcr, wxWindow* parent, wxWindowID id = wxID_ANY,
         const wxPoint& pos = wxDefaultPosition,
         const wxSize& size = wxDefaultSize,
         long style = wxScrolledWindowStyle,
         const wxString& name = wxPanelNameStr);
-    virtual ~FieldBrowser() override = default;
+    virtual ~FieldThumbnailBrowser() override = default;
 
-    void OnCmdHelp(wxCommandEvent& event);
     virtual void OnUpdate(); // Refresh all window controls
 
 private:
-    void CreateControls();
-    void OnChar(wxKeyEvent& event);
     void OnPaint(wxPaintEvent& event);
-    wxSize SizeOfOneCell() const;
-    int WhichCell(wxPoint const& p) const;
     void HandleKey(wxKeyEvent& event);
     void HandleMouseDown(wxMouseEvent& event);
 
+    wxSize SizeOfOneCell() const;
+    int WhichCell(wxPoint const& p) const;
+
     CalChartDoc* mDoc;
-    std::unique_ptr<FieldBrowserView> mView;
-    std::vector<FieldBrowserPerCont*> mPerCont;
+    std::unique_ptr<FieldThumbnailBrowserView> mView;
+    std::vector<FieldThumbnailBrowserPerCont*> mPerCont;
```

Comment:
Looks like this variable is actually unused? Maybe we should just get rid of it.

Reply:
Let me check.  I know the mView is used, but maybe not the vector.

Reply:
Yes, the vector is not used.